Archive/Imo/Imo2021Q1.lean

```diff
@@ -99,28 +98,22 @@ theorem exists_finset_3_le_card_with_pairs_summing_to_squares (n : ℕ) (hn : 10
   · simp only [Finset.mem_insert, Finset.mem_singleton]
     rintro d (rfl | rfl | rfl) <;> constructor <;> linarith only [hna, hab, hbc, hcn]
 
-end Imo2021Q1
-
-open Imo2021Q1
-
 theorem imo2021_q1 :
     ∀ n : ℕ, 100 ≤ n → ∀ A ⊆ Finset.Icc n (2 * n),
```

Comment:
```suggestion
theorem imo2021_q1 (n : ℕ) (hn : 100 ≤ n) {A : Finset ℕ} (hA : A ⊆ Finset.Icc n (2 * n)) :
```

Reply:
I think this was intentional in the original version.

Reply:
Hm, fair enough. I guess making the variables unnamed does make this resemble the problem statement closer.